staging/src/k8s.io/api/core/v1/types.go

```diff
@@ -4298,6 +4298,15 @@ type PodStatus struct {
 	// +featureGate=DynamicResourceAllocation
 	// +optional
 	ResourceClaimStatuses []PodResourceClaimStatus `json:"resourceClaimStatuses,omitempty" patchStrategy:"merge,retainKeys" patchMergeKey:"name" protobuf:"bytes,15,rep,name=resourceClaimStatuses"`
+
+	// TotalRequestedResources is the effective resource requests for this pod taking into consideration init
```

Comment:
Who sets this?  Kubelet?  Scheduler?  Is this field explicitly unset until the pod is scheduled and the node performs an update?  If so, we should describe in this doc when the field is not set.

Reply:
The scheduler sets it upon scheduling failure, the intent there was to allow users to be able to inspect the pod to identify the requested resources since its more difficult to calculate with sidecar init containers.  Otherwise, kubelet owns the value.

Reply:
Modified to be set upon pod creation and then kubelet can update it from there.